### ISSUE TITLE
My application not show in Amplify Console

### ISSUE THREAD

**Issue by nguyenthihaitk** [aws-amplify/amplify-hosting#2549]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-console/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

App name: uz-front-member-moripa

### Region

ap-northeast-1

### Amplify Console feature

Access control

### Describe the bug

I don't see my application[uz-front-member-moripa]  anymore in amplify console. I remember that i didn't delete anymore.
https://ap-northeast-1.console.aws.amazon.com/amplify/home?region=ap-northeast-1#/

What should i do.
2 week before, i have the same issue. Please help!

Thanks

### Expected behavior

my application appear in amplify console.

### Reproduction steps

![image](https://user-images.githubusercontent.com/75294208/151496947-b901853f-5f1a-417d-9ddb-355e035c996d.png)

**Comment by swaminator** [comment]
@nguyenthihaitk are you sure you're looking at the right AWS region?

**Comment by nguyenthihaitk** [comment]
today, more one Amplify application not show in Amplify console.
Region: Tokyo
Application name: uz-front-member-suigeitei
note that each region default max 25 application. But we request to increase quotas to 300.

**Comment by nguyenthihaitk** [comment]
@swaminator 
yes am sure right region!

**Comment by hloriana** [comment]
@nguyenthihaitk What's the application ID? Try running this command to find it: https://docs.aws.amazon.com/cli/latest/reference/amplify/list-apps.html

**Comment by nguyenthihaitk** [comment]
Hi @hloriana 
below the applications are not displaying.
1.   App ID: ddgmed7h1b6ts
      App Name: uz-front-member-moripa
2. App ID: dowdj0k9jpet2
     App Name: tsunagri-front-ec-furutorie-nakamura
3. App ID: du9i89ylccqk2
     App Name: tsunagri-front-ec-onedropfarm
4. App ID: dbsywzyneh1n2
    App Name: uz-front-member-suigeitei

Could you check please.
Thanks

**Comment by nguyenthihaitk** [comment]
@swaminator @hloriana 
Hi support team,

I'm trying to remove Basic authentication, but when I checked the Amplify Console screen, it seems that the top page of TSUNAGRI (tsunagri.com) disappeared.
APP ID: d803qk67ysiqo
App Name: tsunagri-top
This is important application. So, could you check and resolve this as soon as possible please!

Thanks

**Comment by McLeroy** [comment]
I have this same issue.
I used to have 5 apps on the Amplify Console. Just yesterday I logged in and all my apps were gone.
I even tried creating a new app but immediately I navigate to "All apps" the app is gone. Yes, I know Amplify is region specific. I created the app in "us-east-1" and tried to query it in "us-east-1". 
I have attached a drive video link to better explain.

[https://drive.google.com/file/d/1IGwUMN8XDFDRkWZzTrXM83NC5PTXdY6Y/view?usp=sharing](url)
